packages/core/utils/src/relativeUrl.js

```diff
@@ -1,7 +1,10 @@
-// @flow
+// @flow strict-local
+
 import path from 'path';
 import url from 'url';
 
 export default function relativeUrl(from: string, to: string): string {
-  return url.format(url.parse(path.relative(from, to)));
+  return url.format(
+    url.parse(path.posix.relative(from, to).replace(/^\.\.\//, '')),
```

Comment:
Before this change, `relativeUrl('foo/bar', 'foo/baz')` resulted in `../baz`, rather than simply `baz`.